exporting/init_connectors.c

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include "exporting_engine.h"
+#include "graphite/graphite.h"
+
+/**
+ * Initialize connectors
+ *
+ * @param engine an engine data structure.
+ * @return Returns 0 on success, 1 on failure.
+ */
+int init_connectors(struct engine *engine)
+{
+    for (struct connector *connector = engine->connector_root; connector; connector = connector->next) {
```

Comment:
I am sorry, I do not remember, did we agree that we would allow variable declaration inside `for` loop?

Reply:
Next file for example, you are creating the variable outside.

Reply:
Yes, we agreed that we will allow all c99 features in our code on the last meetup.

Reply:
Thank you!